SpatialGDK/Source/SpatialGDK/SpatialGDK.Build.cs

```diff
@@ -23,9 +23,10 @@ public SpatialGDK(ReadOnlyTargetRules Target) : base(Target)
         PublicIncludePaths.Add(WorkerSDKPath); // Worker SDK uses a different include format <improbable/x.h>
         PrivateIncludePaths.Add(WorkerSDKPath);
 
-        PublicDependencyModuleNames.AddRange(
+		PublicDependencyModuleNames.AddRange(
```

Comment:
Spacing seems off

Reply:
Changed the whole file from spaces to tabs.